src/app.ts

```diff
@@ -43,10 +44,15 @@ export class App {
     return this.releaseInfo.source_hash
   }
 
+  get sourceMap() {
+    return this.releaseInfo.source_map
```

Comment:
Is there a convention for snake case vs camel case (I'm genuinely curious)? do we only use snake case for JSON going to web app?

Reply:
Yes, Ruby's convention is snake_case and since that's what the API is written in, that's the data we receive and need to send.

At some point we should figure out a way to homogenize this. I've never knew what to do in these cases, so I'm just following each language's conventions.